FDD_RF_Module/FDD_RF_Modeling.py

```diff
@@ -67,6 +67,19 @@ def CDDR_tot(self, Real_label, Pred_label):
         CDDR_tot = CD / CP
         return CDDR_tot
 
+    def TPR_FPR_tot(self, Real_label, Pred_label):
```

Comment:
@JanghyunJK I added this part to enable detection performance evaluation, after we have fault-free data included. For now the output for TPR & FPR will be all 1 & 0.